.github/workflows/python.yml

```diff
@@ -25,3 +25,7 @@ jobs:
       - name: Test installation
         run: |
           kiss_icp_pipeline --version
+      - name: Run unittests
```

Comment:
I will see this on the CI and pretend we have unit tests 😜 Just won't open the tab